server/sonar-server/src/main/java/org/sonar/server/setting/ws/ValuesAction.java

```diff
@@ -95,6 +97,7 @@ public void define(WebService.NewController context) {
           "</ul>")
       .setResponseExample(getClass().getResource("values-example.json"))
       .setSince("6.3")
+      .setChangelog(new Change("7.1", "Settings from sonar.properties are explicitly forbidden"))
```

Comment:
Comment is not exact. It should be "The settings from conf/sonar.properties are excluded from results."